src/components/TokenBalance.test.tsx

```diff
@@ -54,14 +54,357 @@ jest.mocked(getDynamicConfigParams).mockReturnValue({
   showBalances: [NetworkId['celo-alfajores']],
 })
 
-describe('FiatExchangeTokenBalance and HomeTokenBalance', () => {
+describe('FiatExchangeTokenBalance', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+    jest.mocked(getFeatureGate).mockReturnValue(true)
```

Comment:
is this required? this component doesn't use a feature gate right?

Reply:
This started failing when I removed the feature gate.

Reply:
Ah, looks like this is needed for positions to be included. I think we also need this for the describe('FiatExchangeTokenBalance and HomeTokenBalance') section too. I think they currently pass from a stray mock from another describe. We should add that back, because I think running them standalone would fail